src/scripts/start-marklogic-rootless.sh

```diff
@@ -66,6 +66,7 @@ fi
     [[ "${MARKLOGIC_USER}" ]] && echo "export MARKLOGIC_USER=$MARKLOGIC_USER" >>/etc/marklogic.conf
     [[ "${JAVA_HOME}" ]] && echo "export JAVA_HOME=$JAVA_HOME" >>/etc/marklogic.conf
     [[ "${CLASSPATH}" ]] && echo "export CLASSPATH=$CLASSPATH" >>/etc/marklogic.conf
+    [[ "${MARKLOGIC_EC2_HOST}" ]] && echo "export MARKLOGIC_EC2_HOST=$MARKLOGIC_EC2_HOST" >>/etc/marklogic.conf
```

Comment:
Just wondering Why Check OVERWRITE_ML_CONF is not used here. We should probably need to have the same structure like start-marklogic.sh to check if OVERWRITE_ML_CONF set to true then override the /etc/marklogic.conf